paparazzi/src/main/java/app/cash/paparazzi/Snapshot.kt

```diff
@@ -17,13 +17,25 @@ package app.cash.paparazzi
 
 import com.squareup.moshi.JsonClass
 import java.util.Date
+import java.util.Locale
 
 @JsonClass(generateAdapter = true)
 data class Snapshot(
-  val name: String,
+  val name: String?,
```

Comment:
currently, this means successive snapshot calls without a label will no-op on recording, due to the file existence check.  might be good to throw in this case.